config/rapidez/vatvalidation.php

```diff
@@ -10,4 +10,12 @@
     // - When a guest cart is active.
     // Enable this setting to allow the validation to be used anywhere.
     'allow_anywhere' => false,
+
+    // Should validation be forced on everything put into the box?
+    // This will automatically count any VAT id invalid if it's not from one of the supported countries (i.e. EU)
+    // Any exclusions that should still bypass the check can be added to the `force_exclusions` option.
+    'force_validation' => false,
+    'force_exclusions' => [
+        // 'GB',
+    ],
```

Comment:
We should know a list of countries validatable by VIES.
Wouldn't we want to exclude the countries that we know can't even be validated by VIES as we already know that validation would fail no matter what?